src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Ball extends ColorSupplier {
+    private String color;
+    private int number;
+
+    @Override
+    public String toString() {
+        return super.toString();
```

Comment:
what did you see in your console when you printed this balls?
let's change this implementation) 
(we should be able to see that it is a ball, its color and number)